app/models/form/lettings/pages/referral_type.rb

```diff
@@ -0,0 +1,10 @@
+class Form::Lettings::Pages::ReferralType < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = "referral"
```

Comment:
nitpicky, but would it make sense for this ID to be `referral_type` so that it's not confused with the other `referral` page?